lib/galaxy/util/__init__.py

```diff
@@ -862,13 +862,24 @@ def string_as_bool_or_none(string):
 
 def listify(item, do_strip=False):
     """
-    Make a single item a single item list, or return a list if passed a
-    list.  Passing a None returns an empty list.
+    Make a single item a single item list.
+
+    If *item* is a string, it is split on comma (``,``) characters to produce the list. Optionally, if *do_strip* is
+    true, any extra whitespace around the split items is stripped.
+
+    If *item* is a list it is returned unchanged. If *item* is a tuple, it is converted to a list and returned. If
+    *item* evaluates to False, an empty list is returned.
+
+    :type item:             str or list or tuple or types.NoneType
+    :param item:            object to make a list from
+    :param bool do_strip:   strip whitespace from around split items, if set to ``True``
```

Comment:
I'd separate type and description, as done above with `item`. Also s/whitespace/whitespaces/
```
:type do_strip:  bool
:param do_strip: strip whitespaces from around split items, if set to ``True``
```